Shippo/Track.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using Newtonsoft.Json;
+
+namespace Shippo {
+    [JsonObject (MemberSerialization.OptIn)]
+    public class Track : ShippoId {
+        [JsonProperty (PropertyName = "carrier")]
+        public object Carrier { get; set; }
```

Comment:
They probably pass because attributes are still public. Can you try making them private with only getter? If this works, could you please do the same for `TrackingEvent` and `TrackingHistory`? Thanks!